docs/curl/usertx.md

```diff
@@ -1,221 +1,417 @@
         "user_reads": [
 			{
-				"userID": "user1",
+				"user_id": "alice",
 				"version": {
-					"block_num": 19,
-					"tx_num": 0
+					"block_num": 5
 				}
 			}
 		],
 		"user_deletes": [
 			{
-				"userID": "user1"
+				"user_id": "alice"
 			}
 		]
 	},
-    "signature": "aGVsbG8="
+    "signature": "MEUCIFDGfF7deiAexylyN/C1DINgz5TA5CbIB/w+AnjhZYYTAiEAvGtgFWWtWWQaGr4EWo4whcs/+pHhgYMyYeFPha8YRhg="
 }'
 ```
 
+The section `user_deletes` is an array and we can pass as many number of users in it and all will be deleted if they exist.
+The signature on the payload is computed using the following command
+```sh
+./bin/signer -privatekey=deployment/sample/crypto/admin/admin.key -data='{"user_id":"admin","tx_id":"1b6d6414-9b58-45d0-9723-1f31712add04","user_reads":[{"user_id":"alice","version":{"block_num":5}}],"user_deletes":[{"user_id":"alice"}]}'
+```
+**Output**
+```
+MEUCIFDGfF7deiAexylyN/C1DINgz5TA5CbIB/w+AnjhZYYTAiEAvGtgFWWtWWQaGr4EWo4whcs/+pHhgYMyYeFPha8YRhg=
+```
+
+Once the transaction is validated and committed, the submitter of the transaction recevies the following transaction receipt
+```json
+{
+  "response": {
+    "header": {
+      "node_id": "bdb-node-1"
+    },
+    "receipt": {
+      "header": {
+        "base_header": {
+          "number": 7,
+          "previous_base_header_hash": "+KCuJm2x9LzsIBFmAVkW6w/Hmw7IRjl52aZjNI7juR8=",
+          "last_committed_block_hash": "7nEaUCRBiGvmrhbhl7tzupb0K/KjGdS/SSJtCLjvsjA=",
+          "last_committed_block_num": 6
+        },
+        "skipchain_hashes": [
+          "7nEaUCRBiGvmrhbhl7tzupb0K/KjGdS/SSJtCLjvsjA=",
+          "IHP38VOoGoVHAHROXPZJ8ZWG3pZ6rmqQYovVzm5i7rw="
+        ],
+        "tx_merkel_tree_root_hash": "iz84iY89BolcBkVD8okAAqg5duGT/+eivcTO4OcpnNM=",
+        "state_merkel_tree_root_hash": "tAb0KTsd9JgCQ1YZMwYLs2O3wpo0ynA7h22B8pKzBC0=",
+        "validation_info": [
+          {}
+        ]
+     }
+    }
+  },
+  "signature": "MEUCIGWLKxgnxwlznZyBbKbpiFcnbUv32CbkBaFitOxEWnSMAiEAqcNh9pswgvF7f1leWrteozij2zSlG78gcfP5+eAfHhA="
+}
+```
+
+If we query the user, the response would not contain any details associated with the user.
+```sh
+curl \
+     -H "Content-Type: application/json" \
+     -H "UserID: admin" \
+     -H "Signature: MEYCIQDioRVRhtdaLEjFSeCPqrVCCtdwq+hvy7Y+i3cXaqhZ3wIhAK/gmdftR4x0KF3w8V86hSYXPehf/rlO8QcSnU9sFnvC" \
+     -X GET http://127.0.0.1:6001/user/alice | jq .
+```
+**Output**
+```json
+{
+  "response": {
+    "header": {
+      "node_id": "bdb-node-1"
+    }
+  },
+  "signature": "MEUCIDV6UgjPLtM5c+WqQ+Ue5xcKe/w85nAdwwRl7qPqJByVAiEA4RTqXa8Xf8S+O8YPRpFgIHzOCH5jOHkL2jMmLG7Zdpw="
+}
+```
+
 ## Addition/Updation/Deletion of a User
 
-Within a single transaction, we can add a new user, update or delete an existing user. 
+Within a single transaction, we can add a new user, update or delete an existing user.
 
 ## Invalid User Administration Transaction
 
-TODO (subsequent PR)
\ No newline at end of file
+TODO (subsequent PR)
```

Comment:
I think this is good - every empty section should have this or be hidden

Reply:
Added TODO for all empty sections